moveit_kinematics/srv_kinematics_plugin/src/srv_kinematics_plugin.cpp

```diff
@@ -328,12 +328,11 @@ bool SrvKinematicsPlugin::searchPositionIK(const std::vector<geometry_msgs::Pose
     if (error_code.val != error_code.SUCCESS)
     {
       ROS_DEBUG_NAMED("srv", "An IK that satisifes the constraints and is collision free could not be found.");
+      // Debug mode for failure:
+      ROS_DEBUG_STREAM("Request was: \n" << ik_srv.request.ik_request);
```

Comment:
These  should be ROS_DEBUG_STREAM_NAMED to enable filtering with rosconsole.yaml

Reply:
Good point.